src/test/kotlin/no/skatteetaten/aurora/databasehotel/testutils.kt

```diff
@@ -46,3 +46,45 @@ fun createOracleSchema(mangerDataSource: HikariDataSource): HikariDataSource {
     val schemaName = manager.createSchema(username, password)
     return DataSourceUtils.createDataSource(mangerDataSource.jdbcUrl, schemaName, password)
 }
+
+fun DatabaseManager.cleanPostgresTestSchemas(schemas: List<Schema>): List<Exception> {
+    val suppressedExceptions = arrayListOf<Exception>()
+    if (this is PostgresDatabaseManager) {
+        val roleRemoveFailed = arrayListOf<Schema>()
+
+        schemas.forEach {
+            try {
+                permanentlyDeleteSchema(it)
+            } catch (e: Exception) {
+                if (e.message!!.contains("cannot be dropped because some objects depend on it")) {
```

Comment:
Prøve å unngå å bruke !!, litt uheldig å få nullpointer. Man her heller ikke ryddet opp hvis det mangler message for exception